src/controllers/queries/__tests__/index.test.ts

```diff
@@ -3,14 +3,23 @@ import { describe, it } from 'mocha'
 import sinon from 'sinon'
 
 import { Request } from 'express'
+import { InvalidInputError } from '../../../errors.js'
 import { mockLogger } from '../../__tests__/helpers.js'
 import { QueriesController } from '../index.js'
 import { mockIds, toHTMLString, withQueriesMocks } from './helpers.js'
 
+const expiresAtExpectation = new Date(1000 + 7 * 24 * 60 * 60 * 1000)
+
 describe('QueriesController', () => {
   const req = { log: mockLogger } as unknown as Request
+  let clock: sinon.SinonFakeTimers | null = null
+
+  beforeEach(() => {
+    clock = sinon.useFakeTimers({ now: 1000, toFake: ['Date'] })
+  })
 
   afterEach(() => {
+    clock?.restore()
```

Comment:
if clock is always created in before each, why are we checking if it even exists before restoring? Won't it always exist?

Reply:
we know that but the type system doesn't. Hence the `?` to convince it that it's all fine

Reply:
ok